changelog.d/20240109_175058_fahad.khalid.md

```diff
@@ -0,0 +1 @@
+- [Bugfix] Fix security context for vector (by @FahadKhalid210)
```

Comment:
Mark this as a breaking change. Note that the vector container will from now on run in privileged mode.

Reply:
I'm not sure I understand. Are you suggesting an alternate solution?

Reply:
It depends. If it is easily possible to create a new user for Vector during build, then we should. Otherwise, I am ok with removing the configs in this PR.

Reply:
@FahadKhalid210 what do you think?